app/assets/stylesheets/layouts/_landing_page.scss

```diff
@@ -56,8 +50,122 @@
 .l-landing-page__title {
   display: inline-block;
   margin-right: $baseline-unit*2;
+  margin-top: $baseline-unit*4;
+  margin-bottom: 0;
+
+  @include respond-to($mq-m) {
+    margin-bottom: $baseline-unit*2;
+  }
 }
 
 .l-landing-page__register-link {
-  display: inline-block;
+  margin-top: 0;
+  display: none;
+
+  @include respond-to($mq-m) {
+    margin-top: $baseline-unit*7;
+    text-align: right;
+    display: inline-block;
+  }
+}
+
+.l-landing-page__find-adviser {
+  background: linear-gradient(to bottom, white, $background-gradient-color);
+  color: $color-black;
+}
+
+.l-landing-page__why-adviser {
+  @include respond-to($mq-m) {
+    margin-bottom: $baseline-unit*6;
+  }
+}
+
+.l-landing-page__search-filter {
+  background-color: $background-color;
+}
+
+.l-landing-page__search-filter label{
+  font-weight: bold;
+  color: $color-black;
+}
+
+.l-landing-page__advice-reasons {
+  background-color: $background-color;
+  color: $color-black;
+  @include respond-to($mq-m) {
+    padding-bottom: $baseline-unit*6;
+  }
+}
+
+.l-landing-page__search-filter__header {
+  margin-top: $baseline-unit*2;
+  margin-left: $baseline-unit*3;
+  margin-bottom: 0;
+
+  @include respond-to($mq-m) {
+    @include body(24, 28);
+  }
+}
+
+.l-landing-page__trust-banner__header {
+  @include respond-to($mq-m) {
+    margin-top: $baseline-unit;
+  }
+}
+
+.l-landing-page__section-header {
+  @include respond-to($mq-m) {
+    @include body(30, 38);
+  }
+}
+
+.l-landing-page__other-information {
+  color: $color-black;
+}
+
+.l-landing-page__registration{
+  display: none;
+  visibility: hidden;
+
+  @include respond-to($mq-m) {
+    display: block;
+    visibility: visible;
+  }
+}
+
+.advice-reasons__explaination {
+  display: none;
+  visibility: hidden;
+
+  @include respond-to($mq-m) {
+    display: inline;
+    visibility: visible;
+  }
+}
+
+.list-numbers {
+  counter-reset:li;
+  padding-left:0;
+  margin: 0 0 $baseline-unit*2 0;
+}
+
+.list-numbers > li {
+  position:relative;
+  margin:0 0 $baseline-unit $baseline-unit*3;
+  padding:4px 8px;
+  list-style:none;
+}
+
+.list-numbers > li:before {
+  content:counter(li);
+  counter-increment:li;
+  @include body(22,28);
+  color: $color-green-secondary;
```

Comment:
If this moves into its own component, we should create a component-level colour variable for this